Macros/e3 Includes/e3_Classes_Shaman.inc

```diff
@@ -53,10 +55,12 @@ SUB SHM_CharacterSettings
 /if (${Debug}) /echo |- SHM_CharacterSettings ==>
 	/call WriteToIni "${Character_Ini},Shaman,Auto-Canni (On/Off)"
 	/call WriteToIni "${Character_Ini},Shaman,Canni"
+	/call WriteToIni "${Character_Ini},Shaman,Idol"
```

Comment:
I _think_ the only Idol available is Idol of Malo. Is it necessary to have a config entry or could Event_Idol just default to idol of malo?

Reply:
In this era yes. I'm not sure if other idols get added later on or not, so I'm trying to future proof it

Reply:
I dunno if you're ever going to need it, the next idol is in Ro. https://everquest.allakhazam.com/db/spelllist.html?name=idol&type=shm&level=1&opt=And+Higher&expansion=&action=search